app/helpers/accounts_helper.rb

```diff
@@ -6,6 +6,14 @@ def account_input(form)
     end
   end
 
+  def payment_source_link_or_text(account)
+    if current_ability.can?(:manage, account)
```

Comment:
I'm wondering if we should be checking on something a little less restrictive like `:show`. While testing this locally, I have some split accounts, which cannot be managed by an account manager. I'm thinking they should still be able to see it, though; just not manage it. Let me do a little more testing.